docusaurus/docs/iOS/03-guides/03-call-and-participant-state.mdx

```diff
@@ -63,15 +63,32 @@ The following fields are available on the call:
 
 ### Participant State
 
-The `CallParticipant` is the most essential component used to render a participant in a call. It contains all of the information to render a participant, such as audio & video renderers, availabilities of audio & video, the screen sharing session, reactions, and etc. Here's how you iterate over the participants:
+The `CallParticipant` is the most essential component used to render a participant in a call. It contains all of the information to render a participant, such as audio & video renderers, availabilities of audio & video, the screen sharing session, reactions, and etc. Here's how you can subscribe to participants updates:
```

Comment:
The object doesn't really contain the renderers but the tracks. Maybe worth changing to avoid confusion.

Reply:
good catch, fixed